pints/_log_priors.py

```diff
@@ -51,14 +52,14 @@ def __init__(self, a, b):
 
     def __call__(self, x):
         if x[0] < 0.0 or x[0] > 1.0:
-            return -float('inf')
+            return -np.inf
```

Comment:
Interesting https://stackoverflow.com/questions/42315541/difference-between-np-inf-and-floatinf